quarkchain/cluster/master.py

```diff
@@ -782,6 +794,19 @@ def __init__(self, env, root_state, name="master"):
 
         self.__init_root_miner()
 
+    def init_grpc_server(self):
```

Comment:
I think starting grpc server should be outside of `MasterServer`, similar to how we start jsonrpc serverat line 1904.

Reply:
can I put grpc server in jsonrpc file?

Reply:
I don't think it's necessary